samples/02.branding-styling-and-customization/g.send-timeout/README.md

```diff
@@ -0,0 +1,309 @@
+# Sample - Customize send timeout
```

Comment:
This sample has replaced the location of change-locale, which I believe was mistakenly moved. Please change this one to h.